tests/threads/message_counter.nim

```diff
@@ -0,0 +1,38 @@
+import os
+import threadpool
+
+var 
+  chanA: TChannel[string]
+  chanB: TChannel[string]
+
+proc heavyTaskA() {.thread.} =
+  chanA.send("One")
+  sleep(400)
+  chanA.send("Three")
+  sleep(400)
+  chanA.close()
+
+proc heavyTaskB() {.thread.} =
+  sleep(200)
+  chanA.send("Two")
```

Comment:
Yes, thanks, they should be ``chanB``. I'm also realized that the last message "Four" is not received because the channel is closed. I've opened a thread on forum to ask the community whether this behavior should be changed. I'll fix the typos and make further changes based on the feedback on the forum.